extract/tunnel-example.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Copy this script, change the remote user and host,
+# then run it to open a local tunnel to the remote MongoDB service.
+
+##
+## DO NOT COMMIT YOUR COPY OF THIS FILE TO THE REPOSITORY
+##
+
+ssh -Nv -L127.0.0.1:27017:0.0.0.0:27017 remote-user@remote-host.example.org
```

Comment:
I tried running this command but got access denied. So i have to use the option `-i mongo.pem` for  tunneling to work. That step needs to be added here.